syft/frameworks/torch/mpc/falcon/falcon_helper.py

```diff
@@ -21,8 +24,33 @@ def __private_unfold(image, kernel_size, padding):
         return image.unfold(kernel_size, padding)
 
     @staticmethod
-    def xor(value, other):
-        return value + other - (value * 2 * other)
+    def xor(
+        value: ReplicatedSharingTensor, other: Union[int, ReplicatedSharingTensor, torch.LongTensor]
+    ) -> ReplicatedSharingTensor:
+        """
+        Compute the XOR value between value and other.
+        If value and other are both ints we should use the "^" operator.
+
+        Args:
+            value (ReplicatedSharingTensor): RST with ring size of 2 or integer value in {0, 1}
+            other (int): integer with value in {0, 1}
+
+        Returns:
+            The XOR computation between value and other
+        """
+
+        assert (
+            isinstance(value, ReplicatedSharingTensor) and value.ring_size == 2
```

Comment:
Q: This looks kind of *ugly* but this is how black formatted it :\

Reply:
no problem :D